pylint/checkers/dunder_methods.py

```diff
@@ -0,0 +1,148 @@
+        "__or__": "Use | operator",
+        "__radd__": "Use + operator",
+        "__rsub__": "Use - operator",
+        "__rmul__": "Use * operator",
+        "__rmatmul__": "Use @ operator",
+        "__rtruediv__": "Use / operator",
+        "__rfloordiv__": "Use // operator",
+        "__rmod__": "Use % operator",
+        "__rdivmod__": "Use divmod built-in function",
+        "__rpow__": "Use ** operator or pow built-in function",
+        "__rlshift__": "Use << operator",
+        "__rrshift__": "Use >> operator",
+        "__rand__": "Use & operator",
+        "__rxor__": "Use ^ operator",
+        "__ror__": "Use | operator",
+        "__iadd__": "Use += operator",
+        "__isub__": "Use -= operator",
+        "__imul__": "Use *= operator",
+        "__imatmul__": "Use @= operator",
+        "__itruediv__": "Use /= operator",
+        "__ifloordiv__": "Use //= operator",
+        "__imod__": "Use %= operator",
+        "__ipow__": "Use **= operator",
+        "__ilshift__": "Use <<= operator",
+        "__irshift__": "Use >>= operator",
+        "__iand__": "Use &= operator",
+        "__ixor__": "Use ^= operator",
+        "__ior__": "Use |= operator",
+        "__neg__": "Multiply by -1 instead",
+        "__pos__": "Multiply by +1 instead",
+        "__abs__": "Use abs built-in function",
+        "__invert__": "Use ~ operator",
+        "__complex__": "Use complex built-in function",
+        "__int__": "Use int built-in function",
+        "__float__": "Use float built-in function",
+        "__index__": "Use index method",
+        "__round__": "Use round built-in function",
+        "__trunc__": "Use math.trunc function",
+        "__floor__": "Use math.floor function",
+        "__ceil__": "Use math.ceil function",
+        "__enter__": "Invoke context manager directly",
+        "__aiter__": "Use iter built-in function",
+        "__anext__": "Use next built-in function",
+        "__aenter__": "Invoke context manager directly",
+        "__copy__": "Use copy.copy function",
+        "__deepcopy__": "Use copy.deepcopy function",
+        "__fspath__": "Use os.fspath function instead",
+    }
+    name = "unnecessary-dunder-call"
+    priority = -1
+    msgs = {
+        "C2801": (
+            "Unnecessarily calls dunder method %s. %s.",
+            "unnecessary-dunder-call",
+            "Used when a dunder method is manually called instead "
+            "of using the corresponding function/method/operator.",
+        ),
+    }
+    options = ()
+
+    def visit_call(self, node: nodes.Call) -> None:
+        """Check if method being called is an unnecessary dunder method."""
+        if (
+            isinstance(node.func, nodes.Attribute)
+            and node.func.attrname in self.includedict
+        ):
+            self.add_message(
+                "unnecessary-dunder-call",
+                node=node,
+                args=(node.func.attrname, self.includedict[node.func.attrname]),
```

Comment:
```suggestion
                args=(node.func.attrname, self.includedict[node.func.attrname]),
                confidence=HIGH,
```
We can use that confidence as there is not inference.

Reply:
Nice!